docs/PrivacyPolicy.md

```diff
@@ -0,0 +1,115 @@
+# Akita Privacy Policy
+
+This Privacy Policy applies to the Akita Browser Extension developed by the contributors at
+[https://github.com/esse-dev/akita](https://github.com/esse-dev/akita). This Privacy Policy
+addresses the following questions:
+
+1. [What information does Akita collect?](#1-What-information-does-Akita-collect?)
+2. [How does Akita use the information it collects?](#2-How-does-Akita-use-the-information-it-collects?)
+3. [What information does Akita share? (none)](#3-What-information-does-Akita-share?)
+4. [How long does Akita retain your data?](#4-How-long-does-Akita-retain-your-data?)
+5. [Which browser permissions does Akita use?](#5-Which-browser-permissions-does-Akita-use?)
+6. [Where can I find Akita's source code?](#6-Where-can-I-find-Akita's-source-code?)
```

Comment:
These links don't work. Remove the `?` from each and remove the `'` from #6. Markdown doesn't include punctuation in its links